src/lib/cms-content.ts

```diff
@@ -0,0 +1,80 @@
+import axios from 'axios';
+
+export type ContentType = 'Case Study' | 'Course' | 'Lab' | 'PDF';
+
+export interface Lesson {
+  title: string;
+  link: string;
+}
+
+export interface ContentItem {
+  title: string;
+  contentType: ContentType;
+  longDescription?: string;
+  slug: string;
+  externalLink?: string;
+  level?: string;
+  durationHours?: number;
+  shortDescription: string;
+  lessons?: Lesson[];
+  fileDownload?: string;
+}
+
+const itemMap = ({
+  title,
+  contentType,
+  longDescription,
+  slug,
+  externalLink,
+  level,
+  durationHours,
+  shortDescription,
+  lessons,
+  fileDownload,
+}: any): ContentItem => {
+  return {
+    title,
+    contentType,
+    longDescription: longDescription || null,
```

Comment:
any of the fields where a check like this is done, you can prob just give it a default value in the arguments. Up to you